fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/os/AgentOperatingSystem.java

```diff
@@ -26,8 +26,8 @@
  * An agent represents a person in the simulation which is able to use public transport
  * facilities, can walk, or use private or shared vehicles for transportation. An agent does
  * not move until its mapped application provides instructions, such as using public transport
- * or a private car.<br>
- *
+ * or a private car.<br/>
+ * <br/>
```

Comment:
Is this just because of checkstyle? We should maybe reconsider the checkstyle rule at work here

Reply:
Yes, checkstyle. It actually proposed to use `<p>..</p>`. I'm really looking forward to 2030, when we use JDK25 as the default compiler which enables Markdown in Javadoc.